shaptools/netweaver.py

```diff
@@ -160,21 +186,28 @@ def install(
             conf_file (str): Path to the configuration file
             root_user (str): Root user name
             password (str): Root user password
+            cwd (str, opt): New value for SAPINST_CWD parameter
```

Comment:
This argument is used to modify the `SAPINST_CWD` parameter to the `sapinst` tool call.
This argument is optional, so the user can decide whether it will be used or not. The thing is that using this is mandatory in some cases: additional dvds (or software folders) are needed to complete the installation. For example: Netweaver DB, PAS and AAS instances.

But, I agree, I will mention that this parameter will delete all of the files on that path except `start_dir.cd`, and to use with care.
I have not found a better solution yet. We could create a new temporary folder to avoid unwanted issues (sensitive files removed, or similar).

Definetely, this thing needs more considerations!
@stefanotorresi thanks for reminding me that details!

Reply:
@arbulu89 looks good to me, apart from this improvement on the docstring to inform about the file deletion.